### ISSUE TITLE
pages are not working

### ISSUE THREAD

**Issue by marbe11** [cdriget/DomoCharts#2]
Hi, I try to install charts, but the pages do not work. On pages are some files missing.

![2016-02-06 3](https://cloud.githubusercontent.com/assets/17102333/12869594/8dabd6c6-cd24-11e5-92ba-dc684efd76b0.png)
![2016-02-06 5](https://cloud.githubusercontent.com/assets/17102333/12869595/9277c0de-cd24-11e5-9560-877efe7001a6.png)

**Comment by cdriget** [comment]
Hi,

Yes, Images, CSS, and Jquery extensions are missing.

You can find the full package on the french non-official Fibaro forum : http://www.domotique-fibaro.fr/index.php/topic/466-graph-de-temp%C3%A9rature-humidit%C3%A9-consommationsur-nas/

To be able to download files, you need to create an account and introduce yourself by creating a new thread in this section : http://www.domotique-fibaro.fr/index.php/forum/6-nouveau-pr%C3%A9sentez-vous/
Just click on the "Commencer un sujet" button. Then you can write in English, that's OK.

**Comment by marbe11** [comment]
Yes, thank you very much, Im very interesting on your charts, thank you for it!
Please, this is new message
SQLSTATE[HY000] [1045] Access denied for user 'user'@'localhost' (using password: YES) 
Should I add new user to sql? What password or where can I set account to connect to sql?
Martin

**Comment by cdriget** [comment]
It depends on your MySQL configuration...
If you want to use root account into config.inc.php, them you must be sure that credentials are OK. Apparently, given your SQL error, you have wrong credentials.
Or create a new MySQL User, with sufficient privilège (create database, create table, alter table, ...)
Or simply import the content of database.sql into phpMyAdmin to manually create the required tables.

**Comment by marbe11** [comment]
Yes, my error, it works wonderfully, perfect work, thank you very much :-)

**Comment by marbe11** [comment]
Some question more, the energy kode seems do not work properly, the api for energy at my last version of HC2 has no response /api/energy/
![2016-02-07](https://cloud.githubusercontent.com/assets/17102333/12871854/ef636b68-cd8d-11e5-93ab-94234871375d.png)

**Comment by cdriget** [comment]
Energy is a little bit tricky to configure, as I did it specifically for the french market, where we have different electricity prices during night & day.
I don't know if it is the same for you, but you need to configure it in the confing.inc.php file
You could just use the following variable value if you have only one constant price :
$TimeHCHP = array(
    '00:00' => 'BASE'
);

Then, please note that energy is updated only once per day, usually at 00:01

**Comment by marbe11** [comment]
Yes, thank, but my error is on Fibaro while reading energy api, the reading from energy api do not work for me.

**Comment by cdriget** [comment]
yes, sorry, I didn't take time to read carefully your screenshot.
It seems that you don't have any device returning power attribute.
Do you have any Wall Plug or devices like that ?

**Comment by marbe11** [comment]
Yes, main AEON energy meter and some other plugin meters.
Here my energy panel on /fibaro/en/consumption/energy.html
![2016-02-07 1](https://cloud.githubusercontent.com/assets/17102333/12875676/a468358c-cdef-11e5-9d06-da9c3fa891d6.png)

**Comment by cdriget** [comment]
OK good.
Can you check if you can see data in the following MySQL tables : domotique_power and domotique_energy_day
Also, can you see power (puissance) charts ?

**Comment by marbe11** [comment]
Yes, I have this chart.
![2016-02-07 2](https://cloud.githubusercontent.com/assets/17102333/12875740/0332d35a-cdf1-11e5-8d83-b06edd566926.png)

**Comment by cdriget** [comment]
OK looks perfect.

Now I suspect that your devices does not report energy measurement.
Can you check the API for those devices using their respective ID ?
http://HC2_IP/api/devices/id_of_device
You should see a properties.showEnergy field.
If not, then the device might not report energy.

**Comment by marbe11** [comment]
Yes, I see this properties and it is set to true. Here is response from /docs/#!/devices/getDevice.
[Response.txt](https://github.com/cdriget/DomoCharts/files/120966/Response.txt)

**Comment by cdriget** [comment]
Yes, it looks normal. That is strange....
In the debug window of you Energy button, I need to see the previous line, which should show the devices list.
That list is then appended to the API URL /api/energy/.../...//compare/devices/power/...
We need to understand why the URL is missing the devices list

**Comment by marbe11** [comment]
Now I have this, I think the API is now working, there are no devices...
[DEBUG] 19:41:01: device => 85 84.0 325.92W
[DEBUG] 19:41:01: device => 88 84.1 75.38W
[DEBUG] 19:41:01: device => 91 84.2 207.84W
[DEBUG] 19:41:01: device => 94 84.3 82.84W
[DEBUG] 19:41:01: device => 98 Elektroohřev žebřík 0.00W
[DEBUG] 19:41:01: device => 153 Elektroměr lednička 1.40W
[DEBUG] 19:41:01: device => 155 Elektroměr kotel 97.60W
[DEBUG] 19:41:01: 85,88,91,94,98,153,155
[DEBUG] 19:41:01: {"success":true,"rowcount":0}
[DEBUG] 19:41:01: Error : Missing data from NAS : /graph/teleinfo_energy_get.php

**Comment by cdriget** [comment]
That's OK now.
Wait 00:00 and you will have new data... because Energy data are collected only once a day.
